modules/root/http_config.ru

```diff
@@ -1,5 +1,10 @@
 require 'root/root_api'
+require 'root/logs_api'
 
 map "/" do
   run Proxy::RootApi
 end
+
+map "/logs" do
```

Comment:
Should probably go into a dedicated module then; I'm ok with delaying this though...